src/stores/betaContext.test.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * @jest-environment jsdom
+ */
+import React, { useEffect } from 'react'
+import { render } from '@testing-library/react'
+import { BetaContextProvider, useBetaContext } from './betaContext'
+
+jest.mock('next/dist/client/router', () => ({
+  __esModule: true,
+  useRouter: () => ({
+    query: {},
+    pathname: '/',
+    asPath: '/',
+    events: {
+      emit: jest.fn(),
+      on: jest.fn(),
+      off: jest.fn(),
+    },
+    push: jest.fn(() => Promise.resolve(true)),
+    prefetch: jest.fn(() => Promise.resolve(true)),
+    replace: jest.fn(() => Promise.resolve(true)),
+  }),
+}))
+describe('beta context', () => {
+  it('expects betaOptIn to be false by default', () => {
+    const TestComponent = () => {
+      const { betaOptIn } = useBetaContext()
+
+      const status = betaOptIn.toString()
+      return <h1>Default beta status is {status}</h1>
```

Comment:
nit: you can also use interpolation to coerce to a string value, ie: ```Default beta status is {`${betaOptIn}`}```
(just if you wanted something a little cleaner :)

Reply:
YEAH, excellent. thank you!